src/gsy_e/gsy_e_core/sim_results/endpoint_buffer.py

```diff
@@ -166,6 +166,19 @@ def _get_future_orders_from_timeslot(future_orders: List, time_slot: DateTime) -
                 for order in future_orders
                 if order.time_slot == time_slot]
 
+    @staticmethod
+    def _get_current_forward_orders_from_timeslot(forward_orders: List,
+                                                  market_time_slot: DateTime,
+                                                  area: "Area") -> List:
+        """Filter orders that have happened in the current simulation time for
+        the specified market time slot."""
+        current_time_slot = area.now
+        last_time_slot = area.now - area.config.slot_length
```

Comment:
Nit:
```suggestion
        previous_time_slot = area.now - area.config.slot_length
```
I'm not sure why, but my brain finds this easier to understand. I would intuitively think that `area.now` is actually the last time slot.
Just a nit anyway.

Reply:
I've actually though of that, but IMO, the favorite word in gsy-e is `last` and not `previous`. I'm telling this because if you search gsy-e or gsy-framwork for the word `previous`, you won't get so many results in code but it is full of `last_*` variables.